proof/README.md

```diff
@@ -4,6 +4,8 @@
 
 Download the SAW binary from the [SAW website](https://saw.galois.com).
 
+The minimum required version of SAW is 3.2.
```

Comment:
todo: double-check this number -- the [latest version of SAW](https://github.com/GaloisInc/saw-script/releases/tag/v1.2) is 1.2.

Reply:
Ha! Good catch. I stupidly put the same number as Cryptol.